modules/vpc/main.tf

```diff
@@ -0,0 +1,99 @@
+
+resource "aws_vpc" "main" {
+  cidr_block = var.vpc_cidr
+}
+
+variable "public_subnet_ids" {
+  default = ["10.0.0.0/24", "10.0.1.0/24"]
```

Comment:
You are relying on defaults too much. Instead, I'd rather remove defaults and pass required variables via l1/main.tf
Same for any other variable that you pass inside the module

Reply:
tried to solve it please check!